_stbt/core.py

```diff
@@ -782,7 +824,12 @@ def detect_motion(self, timeout_secs=10, noise_threshold=None, mask=None):
             imglog.imwrite("absdiff_threshold", thresholded)
             imglog.imwrite("absdiff_threshold_erode", eroded)
 
-            motion = (cv2.countNonZero(eroded) > 0)
+            out_region = _nonzero_region(eroded)
```

Comment:
Not really. "bounding box of two regions" and "bounding box of nonzero pixels"? Or just leave it as is.